tests/positive/general/general.go

```diff
@@ -41,8 +41,40 @@ func init() {
 	register.Register(register.PositiveTest, AppendConfigWithRemoteConfigData())
 	register.Register(register.PositiveTest, VersionOnlyConfig())
 	register.Register(register.PositiveTest, EmptyUserdata())
+	register.Register(register.PositiveTest, VerifyMergingOfMultipleBaseConfigs())
 }
 
+var (
```

Comment:
Because of the additional test-specific data, let's put the new test into a separate file.

Reply:
Sure, let me fix it